wiki/Performance_troubleshooting/en.md

```diff
@@ -10,6 +10,7 @@ Unfortunately, due to the scope of hardware which osu! runs on, it's very hard t
 - Is your frame limiter set to `Unlimited`? Set it to `Optimal` and see if it helps. See [its section below](#unlimited) for more information on why `Unlimited` is usually not the best choice.
 - Are you running windowed? Switch to fullscreen.
 - Have you updated your drivers (especially GPU)? Try doing so and see if things improve. Alternatively, if you recently updated them, try rolling back.
+- Are you using a hybrid graphics setup (such as NVIDIA Optimus)? If that is the case, try running the game on only one of them. For running the game only on the dedicated GPU, you can use a MUX switch if available, or you can connect the computer to an external display.
```

Comment:
```suggestion
- Are you using a hybrid graphics setup such as NVIDIA Optimus? If that is the case, try running the game on only the dedicated GPU, by using a MUX switch if available, or connecting the computer to an external display.
```

Reply:
Many people don't need to run osu! on the dedicated GPU. osu! is capable of running on the integrated GPU, especially for modern iGPU. I can run like more than 500 FPS with my iGPU, which is more than enough.